book.bib

```diff
@@ -1413,4 +1413,14 @@ @Article{ZIBB2018
   langid = {english}
 }
 
+@article{tengeler2020gut,
+  title={Gut microbiota from persons with attention-deficit/hyperactivity disorder affects the brain in mice},
+  author={Tengeler, Anouk C and Dam, Sarita A and Wiesmann, Maximilian and Naaijen, Jilly and Van Bodegom, Miranda and Belzer, Clara and Dederen, Pieter J and Verweij, Vivienne and Franke, Barbara and Kozicz, Tamas and others},
+  journal={Microbiome},
+  volume={8},
+  pages={1--14},
+  year={2020},
+  publisher={Springer}
```

Comment:
would be good to add doi as well. It will create automated link in rendering with certain styles